diff -up libxc-3.0.0/src/Makefile.am.single libxc-3.0.0/src/Makefile.am
--- libxc-3.0.0/src/Makefile.am.single	2016-04-21 11:03:11.000000000 +0200
+++ libxc-3.0.0/src/Makefile.am	2016-04-21 23:31:42.141929318 +0200
@@ -159,13 +159,6 @@ nodist_include_HEADERS += $(XCLIBMODS)
 if COMPILE_SINGLE
 
 if F90_MOD_UPPERCASE
-  LIBSFUNCMOD = LIBXC_S_FUNCS_M.@ax_cv_f90_modext@
-else
-  LIBSFUNCMOD = libxc_s_funcs_m.@ax_cv_f90_modext@
-endif
-nodist_include_HEADERS += $(LIBSFUNCMOD)
-
-if F90_MOD_UPPERCASE
   XCLIBMODS_S = XC_S_F90_LIB_M.@ax_cv_f90_modext@ XC_S_F90_TYPES_M.@ax_cv_f90_modext@
 else
   XCLIBMODS_S = xc_s_f90_lib_m.@ax_cv_f90_modext@ xc_s_f90_types_m.@ax_cv_f90_modext@
