diff -up libxc-2.0.2/src/work_mgga_c.c.orig libxc-2.0.2/src/work_mgga_c.c
--- libxc-2.0.2/src/work_mgga_c.c.orig	2013-07-10 09:53:52.000000000 +0300
+++ libxc-2.0.2/src/work_mgga_c.c	2014-02-17 15:21:02.084447501 +0200
@@ -16,6 +16,8 @@
  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include <string.h>
+
 static void 
 work_mgga_c(const XC(func_type) *p, int np, const FLOAT *rho, const FLOAT *sigma, const FLOAT *lapl, const FLOAT *tau,
 	    FLOAT *zk, FLOAT *vrho, FLOAT *vsigma, FLOAT *vlapl, FLOAT *vtau,
@@ -27,6 +29,9 @@ work_mgga_c(const XC(func_type) *p, int
   FLOAT min_grad2 = p->info->min_grad*p->info->min_grad;
   int ip;
 
+  /* Initialize memory */
+  memset(&r,0,sizeof(r));
+
   r.order = -1;
   if(zk     != NULL) r.order = 0;
   if(vrho   != NULL) r.order = 1;
