diff -up libxc-2.1.2/src/Makefile.am.orig libxc-2.1.2/src/Makefile.am
--- libxc-2.1.2/src/Makefile.am.orig	2015-02-19 14:35:44.000000000 +0100
+++ libxc-2.1.2/src/Makefile.am	2015-04-24 22:46:53.997186406 +0200
@@ -62,11 +62,13 @@ libxc_la_SOURCES = $(libxc_la_FUNC_SOURC
 
 if COMPILE_SINGLE
   nodist_libxc_la_SOURCES = $(libxc_la_FUNC_SINGLE_SOURCES)
+else
+  nodist_libxc_la_SOURCES =
 endif
 
 if COMPILE_FORTRAN
   libxc_la_SOURCES += xc_f.c
-  nodist_libxc_la_SOURCES = libxc_funcs.f90 libxc.f90
+  nodist_libxc_la_SOURCES += libxc_funcs.f90 libxc.f90
 if COMPILE_SINGLE
     nodist_libxc_la_SOURCES += xc_f_s.c libxc_s.f90
 endif
